src/test/scala/latis/input/TestHapiBinaryAdapter.scala

```diff
@@ -0,0 +1,47 @@
+package latis.input
+
+import java.net.URI
+
+import latis.data.{Data, Sample, SeqFunction}
+import latis.metadata.Metadata
+import latis.model._
+import latis.output.TextWriter
+import latis.util.FileUtils
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers._
+
+class TestHapiBinaryAdapter extends FlatSpec {
+
+  val reader = new AdaptedDatasetReader {
+    //def uri: URI = new URI("https://cdaweb.gsfc.nasa.gov/hapi/data?id=AC_H0_MFI&time.min=2019-01-01&time.max=2019-01-02&parameters=Magnitude,dBrms&format=binary")
+    //def uri: URI = FileUtils.resolvePath("data/hapi_binary_data").get.toUri
```

Comment:
No idea why this doesn't work.

Reply:
This appears to be due to latis-data/latis3#33.

Reply:
Ah gotcha.